cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/LocalApplicationManager.java

```diff
@@ -134,8 +133,8 @@
     pipeline.addLast(new ProgramGenerationStage(privilegesManager, authenticationContext));
     pipeline.addLast(new DeleteScheduleStage(programScheduler, scheduler));
     pipeline.addLast(new ApplicationRegistrationStage(store, usageRegistry, ownerAdmin));
-    pipeline.addLast(new CreateSchedulesStage(programScheduler, scheduler));
-    pipeline.addLast(new CreateProgramSchedulesStage(programScheduler, scheduler));
+    pipeline.addLast(new CreateSchedulesStage(programScheduler));
```

Comment:
The stages cleanup is done in a separate PR:
https://github.com/caskdata/cdap/pull/8795